draft-dekater-scion-controlplane.md

```diff
@@ -1948,12 +1948,32 @@ Authentication of SCMP packets is not specified here. In current deployments it
 
 As described previously, the goal of SCION’s beaconing process in the control plane is to securely discover and disseminate paths between any two ASes. This section describes security considerations for SCION's Control Plane that focuses on *inter*-domain routing. SCION does not provide intra-domain routing, nor does it provide end-to-end payload encryption so these topics lie outside the scope of this section.
 
-This section focuses on three kinds of security risks in the control plane:
+This section discusses three kinds threats to the control plane:
 
 1. When an adversary controls one or all core ASes of an ISD and tries to manipulate the beaconing process from the top down (see [](#topdown-manipulate)).
 2. When "ordinary" (non-core) adversaries try to manipulate the beaconing process (see [](#manipulate-beaconing)).
 3. Denial of Services (DoS) attacks where attackers overload different parts of the infrastructure (see [](#dos-cp)).
 
+
+## Assumptions
+
+The following assumptions relate to these security considerations. It is assumed that:
+
+1. Every ISD contains at least one core AS.
+2. ISDs are not internally partitioned, so that each AS can reach every other AS within its ISD.
+3. Parent-Child links form a unidirectional hierarchical topology without cyclic relationships.
+4. An AS is described as 'honest' if its private keys are unknown to the attacker and it uses a unique interface identifier for each link. An honest path is one that only traverses honest ASes. A honest segment is the one created by an honest AS.
+
+## Security Properties {#security-properties}
+
+The properties to be provided by the SCION control plane are:
+
+- Connectivity - For every pair of honest ASes X and Y, X will eventually register enough segments to build at least one path (of any length) leading to Y.
+- Path Consistency - For every honest segment registered in any AS, its sequence of AS entries corresponds to a continuous path in the network of inter-domain links and the inter-domain network topology remains unchanged since the segment was first generated.
```

Comment:
I think the sentence is a bit confusing/unclear to me.
I assume there is a split is as follows?

```
For every honest segment registered in any AS:
  - its sequence of AS entries corresponds to a continuous path in the network of inter-domain links
  - the inter-domain network topology remains unchanged since the segment was first generated.
```
Maybe insert a comma or semicolon after "inter-domain links"?

Reply:
Adopted your suggestion